apps/gateway/src/routes/api/v1/conversations/[conversationUuid]/index.ts

```diff
@@ -1,8 +1,7 @@
-import ROUTES from '$/common/routes'
 import { Hono } from 'hono'
 
 import { chatHandler } from './handlers/chat'
 
 export const chatsRouter = new Hono()
 
-chatsRouter.post(ROUTES.Api.V1.Conversations.Chat, ...chatHandler)
```

Comment:
I'm removing this crap, it's worse than having regular string literals even if it causes some minor duplicity